tests/php/test_class_wc-connect-nux.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class WP_Test_WC_Connect_NUX extends WC_Unit_Test_Case {
+
+	public static function setupBeforeClass() {
+		require_once( dirname( __FILE__ ) . '/../../classes/class-wc-connect-nux.php' );
+	}
+
+	public function test_get_banner_type_to_display_dev_jp() {
+		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
+			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
+			'tos_accepted'                           => false,
+			'should_display_after_cxn_banner' => false,
```

Comment:
I don't have a preference, but array's arrows should either all be aligned or all be singled spaced.